frontend/rolecall/src/app/common_components/full-name.pipe.ts

```diff
@@ -0,0 +1,19 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {User} from '../api/user_api.service';
+
+@Pipe({
+  name: 'fullName'
+})
+export class FullNamePipe implements PipeTransform {
+
+  public transform(user: User, ...args: unknown[]): string {
+    if (!user) {
```

Comment:
The arg type is defined as `user: User`, which means it cannot be null or undefined. Remove this if-check?

Reply:
I want the pipe to be null/undefined sturdy, so I changed the signature to transform(user?: User) { ... }

Reply:
Ok cool, in that case let's use `transform(user: User|undefined|null)`.

There are a couple of issues with using `(user?: User)`
* it is not semantically accurate since the parameter is not actually optional, you always have to pass one into a pipe (even if you pass in undefined or null)
* it does not support potential `null`, since `optional` only means it'll be `User` or `undefined`

Reply:
Done.